topics/setup.md

```diff
@@ -6,13 +6,13 @@ Before you begin [creating your first application](create-first-app.md) to work
 1. If you are going to work with shared code or Android-specific code, you can work on any computer with an operating 
    system supported by [Android Studio](https://developer.android.com/studio).  
    If you also want to write iOS-specific code and run an iOS application on a simulated or real device, use a Mac with a 
-   macOS.  
+   macOS. These steps cannot be performed on other operating systems, such as Microsoft Windows. This is due to an Apple requirement.
 2. Install [Android Studio](https://developer.android.com/studio) – version 4.1 or higher.   
```

Comment:
Should I leave 4.1 here because 4.2 is only Beta and Canary yet? (If yes, then I should also update versions of Xcode and Kotlin plugin.

Reply:
Could you ask the question about AS version in the channel? 
From one side, it's not good to recommend only unreleased versions. However, we did this for the first version of the KMM plugin. 
For the other side, if we support only Beta versions, let's say this. I would avoid referring to Canary. Beta is OK.

As to Kotlin plugin version, see my comment below.

As to Xcode, I checked with the team some time ago and they said that 11.3 is OK. We don't have strict correlation.